### DISCUSSION TITLE
Got an Error when run Quasar dev, pleas help me

### DISCUSSION THREAD

**Discussion by w3ba1809** [quasarframework/quasar#15465] [General - Components / Directives / etc]
the issue start happend from one day to another, I can solve the problem, please help me.  this is the log.

 .d88888b.
d88P" "Y88b
888     888
888     888 888  888  8888b.  .d8888b   8888b.  888d888
888     888 888  888     "88b 88K          "88b 888P"
888 Y8b 888 888  888 .d888888 "Y8888b. .d888888 888
Y88b.Y8b88P Y88b 888 888  888      X88 888  888 888
 "Y888888"   "Y88888 "Y888888  88888P' "Y888888 888
       Y8b

 Dev mode.................. spa
 Pkg quasar................ v1.22.5
 Pkg @quasar/app-webpack... v3.7.1
 Pkg webpack............... v5
 Debugging................. enabled

 Configured browser support (>= 88.12% of global marketshare):
 · Chrome for Android >= 109
 · Firefox for Android >= 109
 · Android >= 109
 · Chrome >= 101
 · Edge >= 107
 · Firefox >= 101
 · iOS >= 14.5-14.8
 · Opera >= 91
 · Safari >= 15.4

 App • Running "@quasar/dotenv" Quasar App Extension...
 App • Extension(@quasar/dotenv): Extending quasar.config.js...
undefined:1
undefined
^

[  SyntaxError: Unexpected token u in JSON at position 0

  - JSON.parse

  - quasar-conf-file.js:30 clone
    [POS System VueJS]/[@quasar]/app-webpack/lib/quasar-conf-file.js:30:15

  - quasar-conf-file.js:390 QuasarConfFile.compile
    [POS System VueJS]/[@quasar]/app-webpack/lib/quasar-conf-file.js:390:29

  - task_queues:96 processTicksAndRejections
    node:internal/process/task_queues:96:5

  - dev.js:224 async goLive
    [POS System VueJS]/[@quasar]/app-webpack/lib/cmd/dev.js:224:3

]

**Comment by yusufkandemir** [comment]
[`@quasar/dotenv`](https://www.npmjs.com/package/@quasar/quasar-app-extension-dotenv) has been deprecated:
> This package has been deprecated
> Author message:
> Package no longer supported. Use 'build->env' in quasar.conf.js to do same thing.

**Reply by w3ba1809** [reply]
thank you for your help, but what I need to change in the qusar.conf.js  with this comment **Use 'build->env' in quasar.conf.js**  thanks again for your kind response.

**Reply by yusufkandemir** [reply]
https://quasar.dev/quasar-cli-webpack/handling-process-env#using-dotenv
